src/application/redux/sagas/updater.ts

```diff
@@ -245,9 +260,10 @@ export function* watchUpdateTask(): SagaGenerator<void, UpdateTaskAction> {
   }
 
   // start the asset updater
-  const assetsHashChan = yield* createChannel<{ assetHash: string; network: NetworkString }>();
-  yield fork(assetsWorker, assetsHashChan);
-  yield fork(watchForAddUtxoAction, assetsHashChan); // this will fee the assets chan
+  const assetsChan = yield* createChannel<{ assetHash: string; network: NetworkString }>();
+  yield fork(assetsWorker, assetsChan);
+  yield fork(watchForAddUtxoAction, assetsChan); // this will fee the assets chan
+  yield fork(newPeriodicSagaTask(updateUtxoAssets(assetsChan), 60_000)); // retry update of all utxos' assets every 60s
```

Comment:
instead of running this update periodically, we shoudl only do when user refresh.  (ie. click on Marina Logo or goes into transaction tab)

less moving pieces

Reply:
done